src/WebJobs.Script.ConsoleHost/Arm/Models/ArmArrayWrapper.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace WebJobs.Script.ConsoleHost.Arm.Models
+{
+    public class ArmArrayWrapper<T>
```

Comment:
I'm assuming this class exists to aid serialization, is that correct? Would probably be a good idea to document this class.